backends/tfhe-cuda-backend/cuda/include/bootstrap.h

```diff
@@ -4,7 +4,7 @@
 #include "device.h"
 #include <cstdint>
 
-enum PBS_TYPE { MULTI_BIT = 0, LOW_LAT = 1, AMORTIZED = 2 };
```

Comment:
Here it would make more sense to have MULTI_BIT and CLASSICAL only, and AMORTIZED would be a PBS_VARIANT instead.

Reply:
Indeed!

Reply:
Thinking again, the amortized implementation is in the limbo. I don't think we need to even mention it as a pbs type or variant. The entry point is completely different and it doesn't need the same buffer structure. It will only be used if the user explictly ask for it, by using its entry point. Thus I will just remove the mention to it.